PVPCPlanner/View/Screen/SettingsView.swift

```diff
@@ -15,21 +16,29 @@ struct SettingsView: View {
             Spacer()
             Text("Select the theme")
                 .font(.headline)
-            Picker("Theme color", selection: $selectedTheme) {
+            Picker("Select the theme", selection: $selectedTheme) {
                 ForEach(ThemeMode.allCases, id: \.self) { mode in
                     Text(LocalizedStringKey(mode.rawValue)).tag(mode)
                 }
             }
             .pickerStyle(SegmentedPickerStyle())
             .padding()
+            Text("Select the localization")
```

Comment:
Esto sigue sin estar localizado

Reply:
Aquí pone el string directamente.

Reply:
en iOs es camel case, no snake case
vamos a hora a mezclar?¿

Reply:
Text("Select the localization") -> No
Text(localized: "text_select_locatization") -> Si

Y para esto ... snake case.. para todo lo demás camelCase
y hasta aqui la chapa 👯

Reply:
Para las keys de strings siempre lo he visto en snake_case. camelCase es para las variables y tal